src/main/java/com/balloon_spring_jpa/balloon/entity/FoilBalloon.java

```diff
@@ -0,0 +1,45 @@
+package com.balloon_spring_jpa.balloon.entity;
+
+import com.balloon_spring_jpa.balloon.balloonEnum.FoilBalloonType;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.persistence.*;
```

Comment:
Please don't use * in the import section.

Reply:
done